lib/ansible/modules/files/assemble.py

```diff
@@ -171,16 +171,15 @@ def assemble_from_fragments(src_path, delimiter=None, compiled_regexp=None, igno
     return temp_path
 
 
-def cleanup(path, result=None):
```

Comment:
In order to let the cleanup function use `module.warn()`, I need to expose module to that function, and `result` is no longer needed. So this is pretty much needed as part of making everything use `module.warn()`.